locale/pt-br/knowledge/getting-started/what-is-node-core-verus-userland.md

```diff
@@ -0,0 +1,39 @@
+---
+title: O que é node core vs userland
+date: '2011-08-26T10:08:50.000Z'
+tags:
+  - npm
+  - core
+  - userland
+  - terminology
+difficulty: 1
+layout: knowledge-post.hbs
+---
+
+Ocasionalmente, nas trocas de mensagens nas listas de discussões do Node.js e nos canais do IRC, você pode notar coisas denominadas como "node-core" e "userland".
```

Comment:
I think it's easier to the reader to not translate `node-core` or `userland` on this doc.